app/build.gradle

```diff
@@ -30,7 +30,6 @@ repositories {
 String[] archs = ['arm64-v8a', 'armeabi', 'mips', 'mips64', 'x86', 'x86_64']
 
 dependencies {
-
```

Comment:
Please don't commit files you havn't touched

Reply:
Can you remove this and force push

Reply:
@mhutti1 Thanks for reviewing. I'll do force push soon after making all the changes that are required.